web/client/components/map/openlayers/Map.jsx

```diff
@@ -37,7 +37,8 @@ var OpenlayersMap = React.createClass({
         changeMeasurementState: React.PropTypes.func,
         registerHooks: React.PropTypes.bool,
         interactive: React.PropTypes.bool,
-        onInvalidLayer: React.PropTypes.func
+        onInvalidLayer: React.PropTypes.func,
+        bbox: React.PropTypes.object
```

Comment:
Since we don't use the bbox, apart from the rotation, I would only have a rotation property